docs/Analyze a stack trace.md

```diff
@@ -0,0 +1,50 @@
+# 스택 추적 분석하기
+
+```
+목차
+외부 소스의 오픈 스택 추적
+새로운 스택 추적을 위한 클립보드 모니터링
+```
+
+앱을 디버깅하려면 종종 스택 추적 작업이 필요합니다. 오류나 예외로 인해 앱이 중단 될 때마다 스택 추적이 이루어집니다. [Thread.dumpStack()](https://developer.android.com/reference/java/lang/Thread.html?hl=ko#dumpStack())과 같은 메소드를 사용하여 앱 코드의 어느 시점에서나 스택 추적을 나오게 할 수 있습니다.
+
+연결된 장치에서 앱이 디버그 모드로 실행되는 동안 Android Studio는 그림1과 같이 **logcat** view에서 스택 추적을 나타내고 강조 표시합니다.
+
+스택 추적에는 호출이 발생한 파일 이름 및 줄 번호와 함께 예외가 발생하는 메소드 호출 목록이 표시됩니다. 강조 표시된 파일이름을 클릭하여 파일을 열고 메소드 호출 소스를 검사 할 수 있습니다. **Up the stack trace**와 **Down the stack trace**를 클릭하여 logcat 창에 표시된 스택 추적 line 사이들을 빠르게 이동합니다.
+
+![img](https://developer.android.com/studio/images/debug/logcat-stacktrace_2x.png?hl=ko)
```

Comment:
You are using hyperlinks of developert.apple.com. In this case, If the paths of links is changed, all images will be disappeared in our markdown files. I have made the Image directory in docs/Debugging. You should add in Image directory and change the paths of image sources just like `./Images/<:image source name>` after download the image sources.

EX) `![](./Images/instruments_window_2x-99f5e667-de70-44c2-96f1-800b8543bc42.png)`